app/assets/images/sephora_style_guide/svgs/flags/korea.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you compress this file with https://jakearchibald.github.io/svgomg/? This will help to prettify markup as well

Reply:
Thanks @kieudongoctruc . There was an decrease! From 11.6k → 11.48k